package.json

```diff
@@ -8,6 +8,8 @@
     "@mdx-js/react": "^v1.6.22",
     "@sentry-internal/global-search": "0.0.43",
     "@sentry/gatsby": "^7.15.0",
+    "@sentry/node": "^7.15.0",
+    "@sentry/profiling-node": "^0.0.0-alpha.1",
```

Comment:
Arf, can you make sure you install 0.0.0-alpha.6 or bump this to .7 (I'll cut a release this morning with some changes)

Reply:
yes can do!